py/client/pydeephaven/session.py

```diff
@@ -181,20 +249,26 @@ def close(self) -> None:
     def release(self, ticket):
         self.session_service.release(ticket)
 
-    def _parse_script_response(self, response):
-        if response.changes.created:
-            for t in response.changes.created:
-                t_type = None if t.typed_ticket.type == '' else t.typed_ticket.type
-                self._tables[t.field_name] = (t_type, Table(session=self, ticket=t.typed_ticket.ticket))
```

Comment:
Looking at this old code, I actually think it was wrong. `field_name` by itself is *not* enough to uniquely distinguish fields - you need `(t.application_id, t.field_name)` or `t.ticket_ticket.ticket`. I would suggest `t.ticket_ticket.ticket` as it's more future-proof.

This also suggests that the documentation to `open_table(self, name: str)` should be updated to mention that it's opening a globally scoped table.

Reply:
Is it guaranteed that globally scoped tables use the `"s/tablename"` format for their tickets? If not, we probably need to use `(t.application_id, t.field_name)` so that we can look up tables in `open_table`.

Reply:
We can store the full field ` _fields`, and search for `f.application_id == "scope" and f.field_name == name`.

Reply:
We should not make any assumptions about the format of `t.typed_ticket.ticket`